src/Popover/Popover.js

```diff
@@ -88,7 +100,9 @@ class Popover extends Component {
                 ...controlProps,
                 tabIndex: 0,
                 role: 'button',
-                'aria-haspopup': true,
+                'aria-controls': newPopperProps.id,
+                'aria-expanded': this.state.isExpanded,
+                'aria-haspopup': !!type ? type : true,
```

Comment:
Is this right? The positive/first ternary return type seems like it could end up being a string.
I imagine the intention is a boolean, but when should this be `false`? (there should probably be a test for this)